README.md

```diff
@@ -159,7 +159,7 @@ This section presents the `Rx` API in its entirety. Let's start with the referen
 
     Dynamically switch between different `Rx`s according to the given
     function, applied on each element of this `Rx`. Each switch will cancel
-    the subscriptions for the previous outgoing `Rx`, and lunch a subscriptions
+    the subscriptions for the previous outgoing `Rx`, and launch the same subscriptions
```

Comment:
Maybe it should be `and lunch a new subscription on the next Rx`. It's not the same, it's really a new one!

For instance

```scala
rx1.flatMap(_ => rx2.fold(0)(_ + _))
```

Will systematically "reset" the `fold` on every `rx1` update.